robot/web/web_c.go

```diff
@@ -166,10 +175,11 @@ func (svc *webService) makeStreamServer(ctx context.Context) (*StreamServer, err
 	}
 
 	addStream := func(streams []gostream.Stream, name string, isVideo bool) ([]gostream.Stream, error) {
-		config := *svc.opts.streamConfig
-		config.Name = name
+		config := gostream.StreamConfig{
```

Comment:
This was the code path for components that existed on viam-server startup. I changed the behavior from:
* Make a copy of the stream config
* Nil out what we don't want

To instead:
* Make a blank config
* Copy the encoder factories we do* want over

I think the latter flows better and makes this consistent with the code change I had for "new cameras".